test/e2e/apimachinery/webhook.go

```diff
@@ -79,6 +73,8 @@ const (
 var serverWebhookVersion = utilversion.MustParseSemantic("v1.8.0")
 
 var _ = SIGDescribe("AdmissionWebhook", func() {
+	defer GinkgoRecover()
```

Comment:
Just curious. Why is it needed here?

Reply:
Its not. While I was working through fixing this I was seeing an error which was being swallowed by the test harness. This allows for the failure to be seen. Creating a new version with this removed.